agent/api/container/container.go

```diff
@@ -1360,3 +1367,39 @@ func (c *Container) IsContainerTornDown() bool {
 	defer c.lock.RUnlock()
 	return c.ContainerTornDownUnsafe
 }
+
+func (c *Container) SetContainerHasPortRange(containerHasPortRange bool) {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+	c.ContainerHasPortRange = containerHasPortRange
+}
+
+func (c *Container) GetContainerHasPortRange() bool {
```

Comment:
Nit - i think it's more common to name a predicate `IsXXX` or `HasXXX`. In this case `HasPortRange` seems a bit easier to read. Imagine we have container c, and caller would call `c.HasPortRange()` instead of `c.GetContainerHasPortRange()`

Reply:
made the change